docs/samples/Microsoft.ML.Samples/Dynamic/OnnxTransform.cs

```diff
@@ -0,0 +1,121 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Transforms;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.ImageAnalytics;
+using Microsoft.ML;
+using System;
+using System.IO;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    class OnnxTransformExample
+    {
+        public static void OnnxTransformSample(string[] args)
+        {
+            // Download the squeeznet image model from ONNX model zoo, version 1.2
+            // https://github.com/onnx/models/tree/master/squeezenet
+            var model_location = @"squeezenet\model.onnx";
+
+            var env = new MLContext();
+
+            // Use the utility functions to inspect models inputs, outputs, shape and type
+            // Load the model using the OnnxModel class
+            var onnxModel = new OnnxModel(model_location);
+
+            // This model has only 1 input, so inspect 0th index for input node metadata
+            var inputSchema = onnxModel.ModelInfo.InputsInfo[0];
```

Comment:
>inputSchema [](start = 16, length = 11)

I wouldn't call this inputSchema, b/c this is not ML.NET schema and we don't want to give that impression. inputInfo would be a better name.